cmd/tsconnect/common.go

```diff
@@ -149,7 +149,11 @@ func runEsbuildServe(buildOptions esbuild.BuildOptions) {
 	if err != nil {
 		log.Fatalf("Cannot start esbuild server: %v", err)
 	}
-	log.Printf("Listening on http://%s:%d\n", result.Host, result.Port)
+	log.Printf("Listening on http://%s:%d\n  - type ctrl-c to stop the dev server.", result.Host, result.Port)
+	for true {
```

Comment:
A shorter way to write `for true {` is just `for {`.

But a shorter way to write these three lines (to block forever) is just `select {}`

Reply:
I'm not a go developer but I think the code works fine and is clear as described.  a random select{} call is much less so. feel free to merge this as-is or write your own to replace it if you offended by my code.   it fixes a real bug.